charts/rancher-webhook/templates/deployment.yaml

```diff
@@ -14,7 +14,7 @@ spec:
       volumes:
       - name: tls
         secret:
-          secretName: rancher-webhook-tls
+          secretName: cattle-webhook-tls
```

Comment:
Why the `cattle` in the secret name? I am not seeing it in previous webhook versions, it's always `rancher-webhook-tls`.

Reply:
https://github.com/rancher/webhook/blob/release/v0.3/pkg/server/server.go#L30